testhelpers/image_privileges.go

```diff
@@ -0,0 +1,35 @@
+package testhelpers
+
+import (
+	"regexp"
+	"strings"
+)
+
+var urlRegex = regexp.MustCompile(`v2\/(.*)\/(?:blobs|manifests|tags)`)
+
+type ImagePrivileges struct {
+	readable  bool
+	writeable bool
+}
+
+func NewImagePrivileges(imageName string) *ImagePrivileges {
+	var isReadable, isWriteable bool
+	if strings.Contains(imageName, "readable") {
+		isReadable = true
+	}
+	if strings.Contains(imageName, "writable") {
+		isWriteable = true
+	}
+	return &ImagePrivileges{readable: isReadable, writeable: isWriteable}
+}
+
+func extractImageName(path string) string {
```

Comment:
Can you please add a description to this function?

Reply:
Added in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/714d0d197edf3ff7391210592d4c3552cdfca10b)